Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -3426,6 +3420,39 @@ private void HandleRegistrationChangeMessage(string sessionId, string userIdCSV)
             UnregisterPlayers(session.Name, usersToRegister.ToArray());
         }
 
+        private void InformOnDestroy(string sessionName)
+        {
+            Session localSession;
+
+            if (!TryGetSession(sessionName, out localSession) || localSession.ActiveSession == null)
```

Comment:
You can make this cleaner by inlining the definition:

```csharp
if (!TryGetSession(sessionName, out Session localSession) || localSession.ActiveSession == null)
```

Reply:
This is resolved by being implemented as you recommended.

Reply:
This pattern is also being implemented in two other places.